server/client.go

```diff
@@ -2901,6 +2894,19 @@ func (c *client) prunePerAccountCache() {
 	}
 }
 
+// getUserInfo returns the user info for the client that can be used
+// for logging.
+func (c *client) getUserInfo() string {
+	switch {
+	case c.opts.Nkey != "":
+		return fmt.Sprintf("Nkey %q", c.opts.Nkey)
+	case c.opts.Username != "":
+		return fmt.Sprintf("User %q", c.opts.Username)
+	default:
```

Comment:
Since we are looking for an auth user here (maybe rename it that), I don't think default should be cid, we get that in client logging. Maybe N/A or unauthorized?